GeoLib/Grid.h

```diff
@@ -506,8 +502,8 @@ void Grid<POINT>::initNumberOfSteps(std::size_t n_per_cell,
 
 	switch (dim.count()) {
 	case 3: // 3d case
-		_n_steps[0] = sc_ceil(std::cbrt(
-			n_pnts*delta[0]*delta[0]/(n_per_cell*delta[1]*delta[2])));
+		_n_steps[0] = sc_ceil(std::cbrt(n_pnts * (delta[0] / delta[1]) *
+			                            (delta[0] / delta[2]) / n_per_cell));
```

Comment:
> mixed tabs + whitespaces

I thought the rule is: identing by tabs, align by spaces?!

> also, could you please explain why the computation has been changed (and why it has only been changed for n_steps[0])

If the `extensions[0]` (`delta` will be renamed to `extensions`) is very small then computing the square is even smaller. There are some cases the result can not represented by a double. In 3d case `extensions[1]` and `extensions[2]` are also very small. The quotients `extensions[0]/extensions[1]` and `extensions[0]/extensions[2]` are probably rather representable than the result of `extension[0]*extension[0]`.

The computation has been changed only in this case because only here a multiplication occurs.